tox.ini

```diff
@@ -1,28 +1,17 @@
 [tox]
-envlist = py26,py27,pypy,py26-trunkdeps,py27-trunkdeps,pypy-trunkdeps
+envlist = py26-{twlatest,tw150},{py26,py27,pypy}-{twtrunk,twlatest,tw150}
```

Comment:
Twisted has (15.3 removed support, 15.4 has unofficial support, 15.5 will refuse to import)